.github/README.md

```diff
@@ -21,3 +21,25 @@ Dette vil automatisk laste inn nye endringer når du lagrer filer, noe som gjør
 
 ### Kjøre testene:
 *Ikke implementert*
+
+### MariaDb database som Docker konteiner
+
+1. Opprett MariaDb konteiner.
+  
+
+```docker
+docker run --name NordicDoor -e MYSQL_ROOT_PASSWORD=PASSORD -p 3306:3306 -d docker.io/library/mariadb:latest
+```
+
+2. Verifiser at konteineren har status 'Running'
+  
+3. Koble til konteiner og logg på som root.
+  
+  ```
+  docker exec -it NordicDoor bash
```

Comment:
Da vil også denne måtte endres

```suggestion
  docker exec -it db bash
```

Reply:
Hva databasen heter spiller ikke så stor rolle for meg, men tenker at det er greit at den har et navn som tilsier at den tilhører NordicDoor applikasjonen.

Hvis den blir endret må vi huske å endre connection string i PDSA_System/Server/appsettings.json

Reply:
Vanligvis vil man ikke ha et så spesifikt navn på container, men heller ha et mer spesifikt navn på "databasen" inni mariadb. Det er godt mulig at de bruker samme sql server til flere ting. Tror hvis det er så viktig å ha `NordicDoor` som navn på en av containerne, så bør det være for .NET-prosjektet, ikke databasecontaineren

En MariaDB container er ***ikke*** begrenset til en applikasjon